common/app/model/Asset.scala

```diff
@@ -107,8 +107,9 @@ case class VideoAsset(
   // The video duration in seconds
   val duration: Int = fields.get("durationSeconds").getOrElse("0").toInt +
                            (fields.get("durationMinutes").getOrElse("0").toInt * 60)
+  val durationSeconds: Int = fields.get("durationSeconds").getOrElse("0").toInt
```

Comment:
On [this point](https://github.com/guardian/frontend/pull/12591/files#r60062191) - Could we not have:
`val durationSeconds: Option[Int] = fields.get("durationSeconds").map(_.toInt)`